modules/social_features/social_embed/src/Plugin/Filter/SocialEmbedUrlFilter.php

```diff
@@ -1,91 +0,0 @@
-<?php
-
-namespace Drupal\social_embed\Plugin\Filter;
```

Comment:
Removing this makes it BC breaking, do we need to?

Reply:
Probably not, as the filter plugin ID remains same 'social_embed_convert_url' and also, its remaining class to be more clear.